src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), color literals like `#f3f3f3` should be replaced with variables for consistency and maintainability. Consider defining a variable for this border color if it's used in multiple places.